src/map.h

```diff
@@ -609,7 +609,8 @@ class map
 // Flags: 3D
     std::string features( const tripoint &p ); // Words relevant to terrain (sharp, etc)
     bool has_flag( const std::string &flag, const tripoint &p ) const;  // checks terrain, furniture and vehicles
-    bool can_put_items( const tripoint &p ); // True if items can be placed in this tile
+    bool can_drop_items( const tripoint &p ) const; // True if items can be dropped in this tile
+    bool can_put_items( const tripoint &p ) const; // True if items can be placed in this tile
```

Comment:
Your newly added method does two things:
* Checks if the item can be placed on the map tile (ie. on terrain or furniture)
* If not, it checks if the item can be placed on the vehicle tile on this map tile

The old method (`can_put_items`) means "can put items on terrain or furniture". Compare `map::move_cost_ter_furn` vs. `map::move_cost`.

User won't see the function itself, but contributors should be able to understand what it does from the name. `can_put` and `can_drop` mean the same thing (actually, `drop` sounds less permissive - `put` suggests that you can spend a while making it "stick"), `can_put_ter_furn` suggests "can put on terrain or furniture but not on anything else" to those who are familiar with `map` class methods.

So it would be then:
* `can_put_items_ter_furn` - can put on terrain or furniture (current `can_put_items`)
* `can_put_items` - can put on anything (current `can_drop_items`)